chanzuckerberg/single-cell-data-portal
# feat: common notification component

## Reason for Change

- #6109 

## Changes

- added common component Notification #6109 and removed redundant code
- added e2e tests #6107 
- activated citation button on page load #6158 
- removed landing notification #6134 
- refactored ShareButton component #5945 

## Testing steps

- Go on rdev [here](https://pr-6269-frontend.rdev.single-cell.czi.technology/)
- Check that Citation is enabled on page load
- Click Citation - notification should pop up and button should be disabled until notification is auto dismissed 
- Click Citation + Share - both notifications should pop up and stack properly
- Share should be disabled as well until notification is auto dismissed

## Checklist 🛎️


## Notes for Reviewer
<img width="1917" alt="Screenshot 2023-11-24 at 2 18 29 PM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/7976579/227de178-f7d3-4183-aa12-7ee5f1f02d4c">
